### PR TITLE
[DOCFIX] Clarify Spark and Alluxio docs

### PR DESCRIPTION
**Pull request by calvinjia**
Clarifies certain behaviors and removes unnecessary lines.

### CODE REVIEW

File: docs/en/Running-Spark-on-Alluxio.md [hunk 1]
```diff
@@ -48,16 +51,45 @@ and the following line to `spark/conf/spark-env.sh`:
...
 
-{% include Running-Spark-on-Alluxio/foo.md %}
+{% include Running-Spark-on-Alluxio/license-local.md %}
 
 Run the following commands from `spark-shell`, assuming Alluxio Master is running on `localhost`:
 
-{% include Running-Spark-on-Alluxio/Alluxio-In-Out-Scala.md %}
+{% include Running-Spark-on-Alluxio/alluxio-local-in-out-scala.md %}
+
+Open your browser and check [http://localhost:19999](http://localhost:19999). There should be an
```
Comment (aaudiber) [inline docs/en/Running-Spark-on-Alluxio.md]: this could be `localhost:19999/browse` so that they don't have to find the tab themselves

**Comment by aaudiber** [comment]
LGTM, left one minor comment. Thanks for improving these docs!

File: docs/en/Running-Spark-on-Alluxio.md (hunk 1 above)
Comment (calvinjia) [inline docs/en/Running-Spark-on-Alluxio.md]: Updated, thanks.

**Comment by aaudiber** [comment]
LGTM

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**